lib/Elastica/Index.php

```diff
@@ -406,7 +424,10 @@ public function addAlias($name, $replace = false)
 
         $data['actions'][] = ['add' => ['index' => $this->getName(), 'alias' => $name]];
 
-        return $this->getClient()->request($path, Request::POST, $data);
+        $endpoint = new Update();
```

Comment:
No, it is in "$data".

Actually here we need to use https://github.com/elastic/elasticsearch-php/blob/master/src/Elasticsearch/Endpoints/Indices/Alias/Put.php, but I don't quickly find a way to work with "$replace" argument there. Any suggestion?